api4/user_test.go

```diff
@@ -802,3 +802,81 @@ func TestResetPassword(t *testing.T) {
 	_, resp = Client.SendPasswordResetEmail(user.Email)
 	CheckBadRequestStatus(t, resp)
 }
+
+func TestGetSessions(t *testing.T) {
+	th := Setup().InitBasic().InitSystemAdmin()
+	defer TearDown()
+	Client := th.Client
+
+	user := th.BasicUser
+
+	Client.Login(user.Email, user.Password)
+	_, resp := Client.GetSessions(user.Id, "")
```

Comment:
Can we switch this to actually get the sessions back, loop through them and verify that the user ID matches?